frontend/src/pages/IstioConfigDetails/IstioObjectDetails/IstioConfigOverview.tsx

```diff
@@ -167,6 +183,14 @@ class IstioConfigOverview extends React.Component<IstioConfigOverviewProps> {
             ></IstioConfigHelp>
           </StackItem>
         )}
+        {isKioskMode() && (
```

Comment:
The `isKioskMode()` func doesn't look like the right func to use here, because that one uses the URL. So, if you navigate or do some action that changes the URL, this block can become hidden even if the user didn't left the Kiosk view.

You should, perhaps, use `isKiosk()` instead of this one. Although, I suggest to wrap this block into a `<KioskElement>` for a cleaner markup (and also to avoid connecting `IstioConfigOverview` to redux), rather than using `isKiosk()`.

Reply:
+1 on <KioskElement>, thanks for the hint